### PR TITLE
Update dependency prettier to ^1.19.1

### PR DESCRIPTION
**Pull request by hadrian-git**
- Updated prettier, mainly to support TypeScript 3.7 syntax
- Ran `yarn prettier` for fixes

Let me know if this should be done in a different way :slightly_smiling_face:

### CODE REVIEW

**Comment by felixfbecker** [comment]
An alternative way to do this (not necessarily better) is to go to https://github.com/sourcegraph/sourcegraph/issues/2422 and force an update of this package:

image

That way it'll automerge if there are no changes needed (otherwise you'll need to check out the branch and commit an update)

**Comment by hadrian-git** [comment]
> An alternative way to do this (not necessarily better) is to go to #2422 and force an update of this package:
> 
> image
> 
> That way it'll automerge if there are no changes needed (otherwise you'll need to check out the branch and commit an update)

Nice! In this case would I still need to checkout the branch for the fixes or would the bot also run them?

**Comment by felixfbecker** [comment]
The bot will not execute Prettier, only update it.

File: browser/src/shared/repo/backend.tsx [hunk 1]
```diff
@@ -34,12 +34,15 @@ export const resolveRepo = memoizeObservable(
...
-                return repository.name
-            }, catchError((err, caught) => caught))
+            map(
+                ({ repository }) => {
+                    if (!repository || !repository.name) {
+                        throw new RepoNotFoundError(rawRepoName)
+                    }
+                    return repository.name
+                },
+                catchError((err, caught) => caught)
```
Comment (felixfbecker) [inline browser/src/shared/repo/backend.tsx]: Wow, this is a bug! This operator is never applied, it's used as the `thisArg` for the map function!

File: browser/src/shared/backend/server.ts [hunk 2]
```diff
@@ -44,5 +47,8 @@ export const fetchSite = (requestGraphQL: PlatformContext['requestGraphQL']): Ob
         mightContainPrivateInfo: false,
     }).pipe(
         map(dataOrThrowErrors),
-        map(({ site }) => site, catchError((err, caught) => caught))
+        map(
+            ({ site }) => site,
+            catchError((err, caught) => caught)
```
Comment (felixfbecker) [inline browser/src/shared/backend/server.ts]: Bug here too!

File: browser/src/shared/backend/server.ts [hunk 3]
```diff
@@ -26,7 +26,10 @@ export const fetchCurrentUser = (
         mightContainPrivateInfo: false,
     }).pipe(
         map(dataOrThrowErrors),
-        map(({ currentUser }) => currentUser || undefined, catchError((err, caught) => caught))
+        map(
+            ({ currentUser }) => currentUser || undefined,
+            catchError((err, caught) => caught)
```
Comment (felixfbecker) [inline browser/src/shared/backend/server.ts]: And here!

File: browser/src/shared/repo/backend.tsx (hunk 1 above)
Comment (hadrian-git) [reply browser/src/shared/repo/backend.tsx]: nice catch

**Pull request merged**